public/jira/history/hooks.ts

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from "react";
```

Comment:
The `jira` folder is meant to be abstract from React.  Could we move this outside somewhere?

Reply:
it is done.  the hooks and components files have been moved and renamed to match the current React strategy.